data/common.yaml

```diff
@@ -9,6 +9,13 @@ profile_update_os::kernel_upgrade::update_minute: 0
 profile_update_os::kernel_upgrade::update_months: []
 profile_update_os::kernel_upgrade::update_week_of_month: ""
 
+profile_update_os::scheduled_reboot::command: "/usr/sbin/reboot"
+profile_update_os::scheduled_reboot::enabled: false
+profile_update_os::scheduled_reboot::reboot_day_of_week: "Mon"
+profile_update_os::scheduled_reboot::reboot_hour: 1
+profile_update_os::scheduled_reboot::reboot_minute: 0
+profile_update_os::scheduled_reboot::reboot_months: "any"
+
 profile_update_os::yum_upgrade::enabled: true
```

Comment:
I know you didn't change this line, but shouldn't this be set to `false` by default?

Reply:
Similarly, we should disable `profile_update_os::yum_upgrade::enabled` by default, correct?

Reply:
I did notice that and thought it was odd, since we have it set to false in our ncsa/control-repo I was going to let it slide, but thinking about it more it's probably best to have it turned off in the module by default as well, so I've set `kernel_upgrade` and `yum_upgrade` both to false